File: examples/js/twitter/trends/getTheCurrentGlobalTrendingTopics.js
const { TwitterService } = require("m3o/twitter");

// Get the current global trending topics
async function getTheCurrentGlobalTrendingTopics() {
  let twitterService = new TwitterService(process.env.MICRO_API_TOKEN);
  let rsp = await twitterService.trends({});
  console.log(rsp);
}

getTheCurrentGlobalTrendingTopics();

File: examples/go/twitter/trends/getTheCurrentGlobalTrendingTopics.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/twitter"
)

// Get the current global trending topics
func GetTheCurrentGlobalTrendingTopics() {
	twitterService := twitter.NewTwitterService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := twitterService.Trends(&twitter.TrendsRequest{})
	fmt.Println(rsp, err)
}
